src/annotation.js

```diff
@@ -37,8 +38,9 @@ export default {
   },
 
   afterDataLimits(chart, args, options) {
-    const state = chartStates.get(chart);
-    adjustScaleRange(args.scale, state, options);
+    if (args.scale.type !== 'category') {
```

Comment:
Category scale can be adjusted outside its labels, so I'm not sure why we exclude it here?

Reply:
Yes  but I can't understand how from the plugin.
Just an example:

```javascript
....
data: {
   labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
   .....
},
options: {
   .....
   plugin: {
     my : {
        ....
        xMax: 'December',
        xMin: 'Feb'
     }
   }
}
```

How you can calculate the min and max? And out to add them? In my opinion, when you have a category scale, you should change min and max because they are not progressive values and the high risk is to have something that you don't want to have on the chart. But Maybe I'm wrong.

Reply:
@kurkle thank you for approving but am I wrong on above thought about category scale? Maybe I'm missing anything

Reply:
You should be able to use numbers relative to label indices. For example, `-0.5` should work.